openweathermap-ios/openweathermap-ios/Maps/ViewControllers/MapViewController.swift

```diff
@@ -89,7 +89,7 @@ extension MapViewController: MKMapViewDelegate {
 
 extension MapViewController: CLLocationManagerDelegate {
     func locationManager(_ manager: CLLocationManager, didChangeAuthorization status: CLAuthorizationStatus) {
-        if status != .authorizedWhenInUse {
+		if status != .authorizedWhenInUse && status != .notDetermined {
```

Comment:
Have you changed the indentation?

Reply:
No.. i think my XCode automatically did it

![Screen Shot 2020-03-28 at 23 43 07](https://user-images.githubusercontent.com/18251220/77838862-24fa6980-714e-11ea-8c07-64ed207f9db3.png)

Is this your XCode ident configuration aswel?

Reply:
The 4 spaces are the default and it is right, but I think you add 1 indentation more by mistake. Check it on your local file because it might be a GitHub issue too.